src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -87,8 +87,12 @@ public function load(array $configs, ContainerBuilder $container)
         // A translator must always be registered (as support is included by
         // default in the Form component). If disabled, an identity translator
         // will be used and everything will still work as expected.
-        if (class_exists('Symfony\Component\Translation\Translator') || $this->isConfigEnabled($container, $config['form'])) {
-            if (!class_exists('Symfony\Component\Translation\Translator')) {
+        if ($this->isConfigEnabled($container, $config['translator']) || $this->isConfigEnabled($container, $config['form'])) {
```

Comment:
We still need it to not always load the `translation.xml` file (in case both the `form` and the `translator` section are not enabled).